salad/operator/scripts/wait_for_epoch_transition.js

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env node
+
+const Web3 = require('web3');
+const {Enigma} = require('enigma-js/node');
+
+const main = async () => {
+    const provider = new Web3.providers.WebsocketProvider(`ws://${process.env.ETH_HOST}:${process.env.ETH_PORT}`);
+    const web3 = new Web3(provider);
+
+    let enigmaHost = process.env.ENIGMA_HOST || 'localhost';
+    let enigmaPort = process.env.ENIGMA_PORT || '3333';
+    const enigmaAddr = process.env.ENIGMA_CONTRACT_ADDRESS;
+    const enigmaTokenAddr = process.env.ENIGMA_TOKEN_CONTRACT_ADDRESS;
+
+    const enigma = new Enigma(
+        web3,
+        enigmaAddr,
+        enigmaTokenAddr,
+        'http://' + enigmaHost + ':' + enigmaPort,
+        {
+            gas: 4712388,
+            from: web3.eth.accounts[0],
```

Comment:
Is this a k8s/compose only script?

Reply:
it should be, yeah. it will only be called when a predeployed set of Salad contracts is not available, which is only true in our testing environments.

Reply:
Sababa